app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -867,6 +870,54 @@ class ComposeActivity :
         binding.addPollTextActionTextView.compoundDrawablesRelative[0].colorFilter = PorterDuffColorFilter(textColor, PorterDuff.Mode.SRC_IN)
     }
 
+    // Use to pass state from cropImage caller to result function
+    private var cropImageItemOld: QueuedMedia? = null
+    private var cropImageUriNew: Uri? = null
+    private var cropImageFileNew: File? = null
+
+    private val cropImage = registerForActivityResult(CropImageContract()) { result ->
```

Comment:
small nitpick on code structure: I'd like to have all contracts in one place so maybe move this up to the others